### PR TITLE
Update except.md

### PR DESCRIPTION
**Pull request by Qinjianbo**
The sample sql should be:
SELECT number FROM numbers(1,10) EXCEPT SELECT number FROM numbers(3,8);

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Fix sample sql: 
```
SELECT number FROM numbers(1,10) EXCEPT SELECT number FROM numbers(3,8);
```

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

File: docs/en/sql-reference/statements/select/except.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Here is a simple example that returns the numbers 1 to 10 that are _not_ a part
 Query:
 
 ``` sql
-SELECT number FROM numbers(1,10) EXCEPT SELECT number FROM numbers(3,6);
+SELECT number FROM numbers(1,10) EXCEPT SELECT number FROM numbers(3,8);
```
Comment (tavplubix) [inline docs/en/sql-reference/statements/select/except.md:33]: Why?

**Comment by tavplubix** [comment]
```
dell9510 :) SELECT number FROM numbers(1,10) EXCEPT SELECT number FROM numbers(3,6);

SELECT number
FROM numbers(1, 10)
EXCEPT
SELECT number
FROM numbers(3, 6)

Query id: 7e890bef-fd7e-4c79-8a56-0c33d23454e4

Connecting to localhost:9000 as user default.
Connected to ClickHouse server version 23.9.1 revision 54465.

┌─number─┐
│      1 │
│      2 │
│      9 │
│     10 │
└────────┘

4 rows in set. Elapsed: 0.062 sec. 

dell9510 :) SELECT number FROM numbers(1,10) EXCEPT SELECT number FROM numbers(3,8);

SELECT number
FROM numbers(1, 10)
EXCEPT
SELECT number
FROM numbers(3, 8)

Query id: b96ad5c3-d38c-4a7e-ba93-229c3104e2ac

┌─number─┐
│      1 │
│      2 │
└────────┘

2 rows in set. Elapsed: 0.014 sec. 

```

**Comment by tavplubix** [comment]
The current version is correct:
https://github.com/ClickHouse/ClickHouse/blob/ce8e72e464d85107d0aebfd5cdea0da0276f9648/docs/en/sql-reference/statements/select/except.md?plain=1#L33-L44

File: docs/en/sql-reference/statements/select/except.md (hunk 1 above)
Comment (Qinjianbo) [reply docs/en/sql-reference/statements/select/except.md:33]: Oh, it's my fault! I misunderstood the usage of numbers functions. I have already checked the usage of the numbers function. Now I understand. Thank you for your answer.

**Pull request closed without merging**